pkg/commands/compute/deploy.go

```diff
@@ -624,6 +684,28 @@ func manageExistingServiceFlow(
 		return serviceVersion, err
 	}
 
+	// Validate that we're dealing with a Compute@Edge 'wasm' service and not a
```

Comment:
I realised when dealing with a user who provides their own service, that we never actually validate the service is a compatible wasm service type. So I added that validation step here...